data.sql

```diff
@@ -13,3 +13,41 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUE
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Angemon',date '2005-04-12',1,FALSE,-45);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Boarmon',date '2005-6-07',7,TRUE,20.4);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Blossom',date '1998-10-13',3,TRUE,17);
+
+
```

Comment:
Please modify the `animals` table to meet the following requirements:
 - if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead.
 - all animals got owners assigned as `Sam Smith owns Agumon`.